MIGRATION.md

```diff
@@ -2,6 +2,21 @@
 
 When new releases include breaking changes or deprecations, this document describes how to migrate.
 
+## Migrating to 1.0
+
+- Dagster's legacy APIs, which were marked "legacy" in 0.13.0, have been removed. This includes `@solid`, `SolidDefinition`, `@pipeline`, `PipelineDefinition`, `@composite_solid`, `CompositeSolidDefinition`, `ModeDefinition`, `PresetDefinition`, `PartitionSetDefinition`, `InputDefinition`, `OutputDefinition`, `DynamicOutputDefinition`, `pipeline_failure_sensor`, `@hourly_schedule`, `@daily_schedule`, `@weekly_schedule`, and `@monthly_schedule`. [Here is a guide](https://docs.dagster.io/0.15.6/guides/dagster/graph_job_op) to migrating from the legacy APIs to the stable APIs.
+- `ScheduleExecutionContext` and `SensorExecutionContext` APIs have been removed. In a previous release, these were renamed to `ScheduleEvaluationContext` and `SensorEvaluationContext` respectively.
```

Comment:
@dpeng817 did we go through a deprecation cycle with these?

Reply:
They had been removed from all API docs around the 0.13.0 timeframe and communicated as renamed, but we never threw warnings.

Reply:
Ok I think that's fine - let's make sure to call out that we did the replacement in 0.13.0 "deprecated" them then.